altair/utils/schemapi.py

```diff
@@ -181,19 +240,53 @@ def __str__(self):
             for val in schema_path[:-1]
             if val not in (0, "properties", "additionalProperties", "patternProperties")
         )
-        message = self.message
-        if self._additional_errors:
-            message += "\n        " + "\n        ".join(
-                [e.message for e in self._additional_errors]
-            )
-        return """Invalid specification
 
-        {}, validating {!r}
+        # Output all existing parameters when an unknown parameter is specified
+        if (
+            hasattr(vegalite, schema_path.split(".")[-1])
+            and self.validator == "additionalProperties"
```

Comment:
I had to add this line to restrict this to only activate for unknown parameter names and not also unknown values to existing parameter names.